article/_posts/2021-01-14-MedAL.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: review
+title: "MedAL: Accurate and Robust Deep Active
+Learning for Medical Image Analysis"
+tags: active-learning deep-learning medical classification
+author: "Thierry Judge"
+cite:
+    authors: "Asim Smailagic, Hae Young Noh, Aure´lio Campilho, Pedro Costa, Devesh Walawalkar, Kartik Khandelwal,
+Mostafa Mirshekari, Jonathon Fagert, Adrian Galdran, Susu Xu"
+    title:   "MedAL: Accurate and Robust Deep Active
+Learning for Medical Image Analysis"
+    venue:   "ICMLA 2018"
+pdf: "https://arxiv.org/ftp/arxiv/papers/1809/1809.09287.pdf"
+---
+
+# Introduction
+
+The authors introduce a novel method for active learning that aims to select more representative samples that standard active learning heuristics. 
+
+# Summary of Contributions
+
+* Novel AL sampling method that maximizes the representativeness of samples selected for labeling.
+* Better initialization by selecting the initial pool with standard feature extraction and description methods
+
+# Existing Methods 
+
+Consider an empty training set $$D_{train}$$ and an unlabeled dataset  $$D_{oracle}$$
+
+Standard Active learning loop
+
+1. Label $n$ samples from  $$D_{oracle}$$ and add them to $$D_{train}$$ .
+2. Train model on $$D_{train}$$ .
+3. Evaluate the model's uncertainty on $$D_{oracle}$$ .
+4. Label $$n$$ most uncertain samples and add them to $$D_{train}$$ .
+5. Repeat steps 2 to 4.
+
+Many heuristics can be used to evaluate uncertainty. 
+
+# Method
+
+## Sampling method
+
+In this work, the authors use rich data descriptions $$f(x) \in R^n$$ to select the most representative sample. This is done by evaluating the average distance with all the samples in $$D_{train}$$. 
+
+
+$$
+S(x) = \frac{1}{N}\sum_{i=1}^N d(f(x_i), f(x))
+$$
+
+
+where $$x_i \in D_{train}$$.  
+
+The rich data descriptor $$f(x)$$ used is a deep representation obtained from a layer in CNN trained for classification. The representations are therefore updated at every step. 
+
+To avoid computing this for each sample, $$s(x)$$ is only computed on the $$M$$ samples with the highest predictive entropy. 
```

Comment:
Could you clarify what is computationally expansive here? Reading your explanation, it seems that the data descriptor is obtained from the feature maps, but it doesn't make it clear if extracting those descriptors is costly. I assume that's what we want to avoid, but the incentive could be better explained.

Reply:
You define `S` and then you use `s`. Are they the same?

Reply:
yes they are the same, I corrected this.